### PR TITLE
Ncnews 4

### PR DESCRIPTION
**Pull request by xtej4ik**

### CODE REVIEW

**Review by Pipe-Smoking-Rabbit — APPROVED** [review]
Hey Inna, it's looking good all the functionality is there! 
I did have trouble accessing the reviews page at first because of the login screen it opens up to, for while you are developing the app please remove the functionality of having to login for now or set a default user as the state. Logging in is something you can easily come back to when more of the app is built :)

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return axios
+    .get(`https://nc-news-p3zx.onrender.com/api/users`)
+    .then((response) => {
+      const result = response.data.users.filter(user => user.username === username);
+
+      if(result.length > 0) {
+        return result[0]
+      }
+
+      return undefined
```
Comment (HeatherEver) [inline src/api.js:25]: have a look at using `axios.create()` to create a baseURL so you don't have to use your long url in each function, keeping things tidy! 
https://notes.northcoders.com/courses/js-front-end/axios

File: src/components/AllArticles.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <h2>All Articles</h2>
+        {isLoading ? (
+        <h3>Loading...</h3>
+      ) : (
+        <ul style={{paddingInline:0}}>
+          {list.map((article) => {
+                return (
+                    <li key={article.article_id} className="article">
+                        <img src={article.article_img_url} alt="article image" className="responsive-img"></img>
+                        <p><h4>Title: {article.title}</h4></p>
```
Comment (HeatherEver) [inline src/components/AllArticles.jsx]: there's a warning in the console about `<h4> cannot be a child of <p>` so just to keep our console free of warnings please change this to either p or h4 :)

**Review by HeatherEver — APPROVED** [review]
Nice work Inna!

**Review by HeatherEver — APPROVED** [review]

**Pull request merged**